ui/src/services/index.ts

```diff
@@ -155,18 +154,19 @@ export async function getSimpleClusterList(): Promise<API.SimpleClusterListRespo
     method: 'GET',
   });
   if (r.successful) {
-    return{
+    return {
       ...r,
-      data:r.data.map((clusterDetail) => ({
-        clusterId: clusterDetail.clusterId,
+      data: r.data.map((clusterDetail) => ({
+        clusterId: clusterDetail.clusterId, // clusterId is not unique
+        id: clusterDetail.name + clusterDetail.namespace,
```

Comment:
Use unambiguous id like `$namespace:$name`. Concatenating the name and namespace with `+` will lead to potential collision. e.g. resource named `ab` in namespace `cd` will has the same id with another resource named `a` in namespace `bcd`.